src/serlo-editor/store/plugin/saga.ts

```diff
@@ -14,7 +14,7 @@ import {
 } from '../documents'
 import { editorPlugins } from '@/serlo-editor/plugin/helpers/editor-plugins'
 
-// TODO: Move all plugin sagas to documents slice and remove plugin slice
+//TODO: Move all plugin sagas to documents slice and remove plugin slice
```

Comment:
@hejtful  should we keep this TODO or turn it into an issue / normal comment?

Reply:
It can be a normal comment, and milder wording, like "Move..." -> "Consider moving..."